src/MCPClient/lib/settings/common.py

```diff
@@ -16,32 +16,186 @@
+LoadSupportedCommandsSpecial = True
+numberOfTasks = 0
+elasticsearchServer = localhost:9200
+disableElasticsearchIndexing = False
+temp_dir = /var/archivematica/sharedDirectory/tmp
+removableFiles = Thumbs.db, Icon, Icon\r, .DS_Store
+    """
+
+    """
+    Mapping configuration attributes with INI section, options and types.
+    """
+    MAPPING = {
+        # [MCPClient]
+        'shared_directory': {'section': 'MCPClient', 'option': 'sharedDirectoryMounted', 'type': 'string'},
+        'processing_directory': {'section': 'MCPClient', 'option': 'processingDirectory', 'type': 'string'},
+        'rejected_directory': {'section': 'MCPClient', 'option': 'rejectedDirectory', 'type': 'string'},
+        'watch_directory': {'section': 'MCPClient', 'option': 'watchDirectoryPath', 'type': 'string'},
+        'client_scripts_directory': {'section': 'MCPClient', 'option': 'clientScriptsDirectory', 'type': 'string'},
+        'load_supported_commands_special': {'section': 'MCPClient', 'option': 'LoadSupportedCommandsSpecial', 'type': 'boolean'},
+        'gearman_server': {'section': 'MCPClient', 'option': 'MCPArchivematicaServer', 'type': 'string'},
+        'number_of_tasks': {'section': 'MCPClient', 'option': 'numberOfTasks', 'type': 'int'},
+        'client_modules_file': {'section': 'MCPClient', 'option': 'archivematicaClientModules', 'type': 'string'},
+        'elasticsearch_server': {'section': 'MCPClient', 'option': 'elasticsearchServer', 'type': 'string'},
+        'disable_search_indexing': {'section': 'MCPClient', 'option': 'disableElasticsearchIndexing', 'type': 'boolean'},
+        'removable_files': {'section': 'MCPClient', 'option': 'removableFiles', 'type': 'string'},
+        'temp_directory': {'section': 'MCPClient', 'option': 'temp_dir', 'type': 'string'},
+        'secret_key': {'section': 'MCPClient', 'option': 'django_secret_key', 'type': 'string'},
+
+        # [client]
+        'db_engine': {'section': 'client', 'option': 'engine', 'type': 'string'},
+        'db_name': {'section': 'client', 'option': 'database', 'type': 'string'},
+        'db_user': {'section': 'client', 'option': 'user', 'type': 'string'},
+        'db_password': {'section': 'client', 'option': 'password', 'type': 'string'},
+        'db_host': {'section': 'client', 'option': 'host', 'type': 'string'},
+        'db_port': {'section': 'client', 'option': 'port', 'type': 'string'},
+        'db_pool_max_overflow': {'section': 'client', 'option': 'max_overflow', 'type': 'string'},
+    }
+
+    def __init__(self):
+        self.config = EnvConfigParser(prefix=self.ENV_PREFIX)
+
+        self.read_defaults()
+        self.read_files()
+
+    def read_defaults(self):
+        self.config.readfp(StringIO.StringIO(self.DBSETTINGS_CONFIG))
+        self.config.readfp(StringIO.StringIO(self.CLIENTCONF_CONFIG))
+
+    def read_files(self):
+        self.config.read([self.DBSETTINGS_FILE, self.CLIENTCONF_FILE])
+
+    def get(self, attr, default=None):
```

Comment:
I think this could be improved in a next iteration to allow better environment strings. E.g. now to match `[MCPClient] watchDirectoryPath = ...` we need `ARCHIVEMATICA_MCPCLIENT_MCPCLIENT_WATCHDIRECTORYPATH`. Far from ideal but it's a good compromise. I'm hoping we can add aliases in the future that are more concise.